src/main/resources/ca/on/oicr/gsi/somnus/ui.js

```diff
@@ -16,21 +23,27 @@ export default function(form, environments, services) {
   ttl.type = "number";
   ttl.value = "1";
   const units = document.createElement("SELECT");
-  for (const [name, multiplier] of [
-    ["days", 86400],
-    ["hours", 3600],
-    ["minutes", 60],
-    ["seconds", 1]
+  for (const [name, value] of [
+    ["minutes", "minutes"],
+    ["hours", "hours"],
+    ["days", "days"],
+    ["seconds", "seconds"]
```

Comment:
Could simplify since name and value are always the same, and could also use the values you already have defined:

```
for (const name in TTL_LOOKUP)
```

(and use `name` in place of `value` below since it's the same)